docs/guide/dev/configuration/file_browse_upload/file_browser_api/README.md

```diff
@@ -0,0 +1,192 @@
+---
+category: file-manager
+order: 80
+url: guide/dev_file_browser_api
+menu-title: File Browser API
+meta-title-short: File Browser API
+---
+<!--
+Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
```

Comment:
Old header. Just please do `ctrl+shift+h` changing `Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.` in the whole project.